src/python/qequlacs/simulator.py

```diff
@@ -42,36 +42,8 @@ def run_circuit_and_measure(self, circuit, **kwargs):
         bitstrings = sample_from_wavefunction(wavefunction, self.n_samples)
         return Measurements(bitstrings)
 
-    def get_expectation_values(self, circuit, qubit_operator, **kwargs):
-        if self.n_samples == None:
-            return self.get_exact_expectation_values(circuit, qubit_operator, **kwargs)
-        else:
-            measurements = self.run_circuit_and_measure(circuit)
-            expectation_values = measurements.get_expectation_values(qubit_operator)
-
-            expectation_values = expectation_values_to_real(expectation_values)
-            return expectation_values
-
-    def get_exact_expectation_values(self, circuit, qubit_operator, **kwargs):
-        if self.n_samples != None:
-            raise Exception(
-                "Exact expectation values work only for n_samples equal to None."
-            )
-
-        expectation_values = []
-        qulacs_state = self.get_qulacs_state_from_circuit(circuit)
```

Comment:
Why would you delete this?

Reply:
My thinking was that it is essentially doing the same as in the base interface. But I see how this method would be more efficient as you're not bothering recasting and such. Will bring it back in a future commit